src/main/java/org/reactome/release/goupdate/GoUpdateStep.java

```diff
@@ -65,11 +68,22 @@ public void executeStep(Properties props) throws SQLException
 			MySQLAdaptor adaptor = getMySQLAdaptorFromProperties(props);
 			this.loadTestModeFromProperties(props);
 			
-			long personID = Long.valueOf(props.getProperty("person.id")).longValue();
-			
+			long personID = Long.parseLong(props.getProperty("person.id"));
```

Comment:
Again in reference to an above comment  (https://github.com/reactome/release-go-update/pull/3/files#r410992715), this should probably be drawing from a property called 'personId'